### PR TITLE
Add fiat currencies support and gitignore, fix asset deletion, readme

### PR DESCRIPTION
**Pull request by FactorizeD**

### CODE REVIEW

File: stonks-app/code/data_utils.py [hunk 1]
```diff
@@ -137,7 +137,7 @@ def calculate_historical_value_in_pln(historical_prices, purchase_df, assets_df,
 
     return (
         historical_prices
-        .loc[:, lambda x: ~x.columns.str.endswith('USD=X')]
+        .loc[:, assets_df.index.tolist()]
```
Comment (FactorizeD) [inline stonks-app/code/data_utils.py:140]: Otherwise currency pairs eg. `PLNUSD=X` would be filtered out

File: stonks-app/code/database.py [hunk 2]
```diff
@@ -33,8 +33,6 @@ def get_firestore_client():
 
 
 def generate_random_purchase_data():
-    df = pd.DataFrame()
```
Comment (FactorizeD) [inline stonks-app/code/database.py:36]: Weren't used before

File: stonks-app/code/database.py [hunk 3]
```diff
@@ -189,6 +193,6 @@ def add_user_purchase_data_to_db(passphrase, data):
 def delete_user_purchase_data(passphrase, id_):
     hash_ = hash_passphrase(passphrase)
     db = get_firestore_client()
-    docs = db.collection(collection_name).where('hash', '==', hash_).where('id', '==', id_).stream()
+    docs = db.collection('purchases').where('hash', '==', hash_).where('id', '==', int(id_)).stream()
```
Comment (FactorizeD) [inline stonks-app/code/database.py:196]: `doc` didn't seem to have a `delete()` method, had to call `.reference` first. As for the `int()`, `id_` is of numpy64 int type which wasn't accepted

**Review by mkusm — APPROVED** [review]
<3

**Pull request merged**